sublime_syntaxes/Assembly x86.sublime-syntax

```diff
@@ -0,0 +1,67 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: Assembly x86 (NASM)
+file_extensions:
+  - asm
+  - inc
```

Comment:
It's generated from the tmLanguage file. I can add it but it seems like it would something that would break when we update.
An alternative would be to only keep the submodule for the official package repo and only have the `.sublime-syntax` files that can be updated manually so it's easier to spot differences like that